### ISSUE TITLE
ember electron:make doesn't fail on forge error

### ISSUE THREAD

**Issue by bendemboski** [felixrieseberg/ember-electron#212]
The easiest way to reproduce this error is to take a working `ember-electron` project and modify the `version` field in your `package.json` to be an invalid semver version, e.g. `123412-dfjlksdfj-124u238o4`. Then run `ember electron:make` and check the return code, and even though it failed, the return code will be `0`.

```
~/projects/eureka$ ember electron:make
bootstrap-sassy config: no JS enabled, glyphicons enabled
WARN: Output exceeds 32000 characters
[WARN] (broccoli-uglify-sourcemap) Minifying: `assets/vendor.js` took: 26991ms (more than 20,000ms)
cleaning up...
Built project successfully. Stored in "/Users/ben/projects/eureka/tmp/electron-build-tmp".

Installing production dependencies into Electron Build folder
npm WARN Invalid version: "123412-dfjlksdfj-124u238o4"
npm WARN electron-build-tmp No description
npm WARN electron-build-tmp No repository field.
npm WARN electron-build-tmp No README data
npm WARN electron-build-tmp No license field.
Making app w/ `electron-forge`
✔ Preparing to Package Application for arch: x64
Unhandled rejection Error: Cannot find module '/Users/ben/projects/eureka/tmp/electron-build-tmp/node_modules/electron-compile/lib/cli.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at _callee2$ (/Users/ben/projects/eureka/node_modules/electron-forge/dist/util/compile-hook.js:181:36)
    at tryCatch (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:64:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:299:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:116:21)
    at GeneratorFunctionPrototype.tryCatcher (/Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/util.js:16:23)
    at PromiseSpawn._promiseFulfilled (/Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/generators.js:97:49)
    at /Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/generators.js:201:15
    at /Users/ben/projects/eureka/node_modules/electron-forge/dist/util/ora-handler.js:70:5
    at Promise.F (/Users/ben/projects/eureka/node_modules/core-js/library/modules/_export.js:35:28)
    at asyncOra (/Users/ben/projects/eureka/node_modules/electron-forge/dist/util/ora-handler.js:69:10)
    at _callee3$ (/Users/ben/projects/eureka/node_modules/electron-forge/dist/util/compile-hook.js:46:45)
    at tryCatch (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:64:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:299:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:116:21)
    at GeneratorFunctionPrototype.tryCatcher (/Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/util.js:16:23)
    at PromiseSpawn._promiseFulfilled (/Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/generators.js:97:49)
    at /Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/generators.js:201:15
    at /Users/ben/projects/eureka/node_modules/electron-forge/dist/util/compile-hook.js:205:17
    at _callee2$ (/Users/ben/projects/eureka/node_modules/electron-forge/dist/api/package.js:269:56)
    at tryCatch (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:64:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:299:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:116:21)
    at GeneratorFunctionPrototype.tryCatcher (/Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/util.js:16:23)
    at PromiseSpawn._promiseFulfilled (/Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/generators.js:97:49)
    at /Users/ben/projects/eureka/node_modules/electron-forge/node_modules/bluebird/js/release/generators.js:201:15
    at packageOpts.afterCopy._regenerator2.default.mark._regenerator2.default.wrap._context3.prev (/Users/ben/projects/eureka/node_modules/electron-forge/dist/api/package.js:280:32)
    at Array.<anonymous> (/Users/ben/projects/eureka/node_modules/electron-packager/common.js:278:13)
    at each (/Users/ben/projects/eureka/node_modules/run-series/index.js:17:24)
    at _callee$ (/Users/ben/projects/eureka/node_modules/electron-forge/dist/api/package.js:244:27)
    at tryCatch (/Users/ben/projects/eureka/node_modules/regenerator-runtime/runtime.js:64:40)

~/projects/eureka$ echo $?
0
```

Interestingly, `ember electron:package` will also fail, but with a return code of `1`. I tried doing the equivalent operations just using the `electron-forge` CLI in a simple project, and both `forge make` and `forge package` had return codes of `1`, so it's something to do with how `ember-electron` is calling `electron-forge`'s `make` API method.

**Comment by anulman** [comment]
Neat! Can look into this shortly. IIRC there's _very_ little difference between how we call `:package` vs `:make`; I'd bet if forge is handling this it's somewhere in their CLI vs. API code.

Regardless, definitely an issue.

**Comment by pichfl** [comment]
Was this resolved by #227?

**Comment by bendemboski** [comment]
Yeah, I guess it was. The underlying problem is probably still there, but as far as I can tell, we can't actually trigger it anymore, so I'll close this issue.
